bin/index.js

```diff
@@ -65,3 +63,5 @@ Please fix the error and try again.
 )
     process.exitCode = 1
   })
+  // remove the blow when https://blinkmobile.atlassian.net/browse/CC-22 is done
+  .then(() => process.listenerCount('SIGINT') > 0 ? process.exit() : true)
```

Comment:
I assume this will still exit with with an exitCode of 1 because of the `process.exitCode = 1` statement in the `.catch()`